doc/source/user/how-to-partition.rst

```diff
@@ -155,7 +155,7 @@ of ``start``) and ends with ``base ** stop``::
   >>> np.logspace(2, 3, num=5, base=2)
   array([4.        , 4.75682846, 5.65685425, 6.72717132, 8.        ])
 
-nD domains
+ND domains
 ==========
 
 nD domains can be partitioned into *grids*. This can be done using one of the
```

Comment:
I looked, and I can't find an authoritative right or wrong. But this is how it seemed to be in a few other places within the NumPy documentation.
```suggestion
N-D domains
===========

N-D domains can be partitioned into *grids*. This can be done using one of the
```